lang/vi/events/github/star.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+return [
+    'created' => [
+        'title' => '🎉 <b>New Star 🌟</b> from 🦑:user',
+        'seeder' => '👤 Starer: <b>:sender_login</b> 👀',
+    ],
+
+    'deleted' => [
+        'title' => '🚫 <b>Star 🌟 Removed</b> from 🦑:user',
+        'remover' => '👤 Remover: <b>:sender_login</b> 👀',
```

Comment:
Please update translation for vi

```suggestion
        'title' => '🎉 <b>Nhận dấu sao mới 🌟</b> từ 🦑:user',
        'seeder' => '👤 Người gắn dấu sao: <b>:sender_login</b> 👀',
    ],

    'deleted' => [
        'title' => '🚫 <b>Dấu sao đã bị hủy <b> từ 🦑:user',
        'remover' => '👤 Người hủy bỏ: <b>:sender_login</b> 👀',
```